dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/util/ESMappingUtilHelperTest.java

```diff
@@ -0,0 +1,570 @@
+     * @param testCase
+     * @param fields
+     * @param expectedResult
+     * @throws DotDataException
+     * @throws IOException
+     * @throws DotSecurityException
+     */
+    @UseDataProvider
+    @Test
+    public void testValidateNewsLikeMapping(final String testCase, final String[] fields, final String expectedResult)
+            throws DotDataException, IOException, DotSecurityException {
+        final ContentType newsContentType = TestDataUtils.getNewsLikeContentType();
+        fieldAPI.save( FieldBuilder.builder(TextField.class).contentTypeId(newsContentType.id()).name("mylatlon").variable("mylatlon").indexed(true).build(), user);
+        Contentlet newsContent = null;
+        try {
+            final String categoryName = "myCategory" + System.currentTimeMillis();
+            Category category = new Category();
+            category.setCategoryName(categoryName);
+            category.setKey(categoryName);
+            category.setCategoryVelocityVarName(categoryName);
+            category.setSortOrder(2);
+            category.setKeywords(null);
+            categoryAPI.save(null, category, user, false);
+            newsContent = new ContentletDataGen(newsContentType.id()).languageId(language.getId())
+                    .host(APILocator.getHostAPI()
+                            .findDefaultHost(APILocator.systemUser(), false))
+                    .setProperty("title", "newsContent Title" + System.currentTimeMillis())
+                    .setProperty("urlTitle", "myUrlTitle")
+                    .setProperty("urlMap", "myUrlMap")
+                    .setProperty("byline", "byline")
+                    .setProperty("sysPublishDate", new Date())
+                    .setProperty("story", "newsStory")
+                    .setProperty("latlong", "[90, -90]")
+                    .setProperty("mylatlon", "[90, -90]")
+                    .setProperty("tags", "test").next();
+
+            newsContent = contentletAPI.checkin(newsContent, CollectionsUtils.list(category), null, user, false);
+            validateMappingForFields(testCase, newsContentType.variable().toLowerCase(), fields, expectedResult);
+        }finally{
+            if (newsContent != null) {
+                ContentletDataGen.destroy(newsContent);
+            }
+        }
+
+    }
+
+    /**
+     * <b>Test Case:</b> Validates dynamic mappings defined in the `es-content-mapping` file are applied correctly<p></p>
+     * <b>Expected Results:</b> Each test case should match the `expectedResult` value, which is a string with the correct datatype in ES
+     * @param testCase
+     * @param fields
+     * @param expectedResult
+     * @throws DotDataException
+     * @throws IOException
+     */
+    @UseDataProvider
+    @Test
+    public void testValidateFileAssetMapping(final String testCase, final String[] fields, final String expectedResult)
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5739222)